app/views/agent/partials/VatReturnsPartial.scala.html

```diff
@@ -22,20 +22,18 @@
 @(mandationStatus: String)(implicit messages: Messages,appConfig: AppConfig)
 <div class="tile tile-no-border">
     <div id="vat-returns" class="tile-body">
-        <h2 class="heading-medium push--top">@messages("vatReturnsPartial.heading")</h2>
+        <h2 class="govuk-heading-m push--top">@messages("vatReturnsPartial.heading")</h2>
         @if(mandationStatus==MandationStatus.nonMTDfB || mandationStatus==MandationStatus.nonDigital || mandationStatus==MandationStatus.MTDfBExempt){
-        <p>@messages("vatReturnsPartial.paragraphNonMandated")</p>
-        <ul>
-            <li class="link-bottom-margin">
-                <a id="submit-vat-returns-link" href="@{appConfig.returnDeadlinesUrl}">@messages("vatReturnsPartial.submitReturnLink")</a>
-            </li>
+        <p class="govuk-body">@messages("vatReturnsPartial.paragraphNonMandated")</p>
+        <p class="govuk-body">
```

Comment:
Can you change the structure of these links back to being a list instead of paragraphs. There is an example of a list of links here https://design-system.service.gov.uk/styles/typography/#lists